### PR TITLE
chmod 600 to config file

### PR DESCRIPTION
**Pull request by csgillespie**

### CODE REVIEW

**Comment by eddelbuettel** [comment]
Good catch.  Not sure why this now failed on Travis CI when it didn't before -- can you poke around.

**Comment by eddelbuettel** [comment]
> Where/how is this file created?

In `.travis.yml`: https://github.com/eddelbuettel/rpushbullet/blob/b66c07cef46ecc1c6858606cdaf4545e745b38fc/.travis.yml#L19-L22


I think the `stopifnot()` is your enemy.  Maybe just skip tests instead of aborting?

**Comment by csgillespie** [comment]
I did know about your PR policy, but I figured a one line PR was just as easy for you to accept/reject as an issue followed by a PR.

I think there's an easy fix for travis

**Comment by eddelbuettel** [comment]
Do you mind if I just squash-merge it?

File: .travis.yml [hunk 1]
```diff
@@ -20,19 +20,19 @@ before_install:
...
   # cf https://docs.travis-ci.com/user/environment-variables/
   - if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ]; then openssl aes-256-cbc -K $encrypted_988d19a907a0_key -iv $encrypted_988d19a907a0_iv -in .rpushbullet.json.enc -out ~/.rpushbullet.json -d; fi
-  
+
 install:
   - ./run.sh install_aptget r-cran-curl r-cran-jsonlite
 
 script:
-  - ./run.sh run_tests
+  - if ["$TRAVIS_PULL_REQUEST" == "false"]; then ./run.sh run_tests; fi
```
Comment (eddelbuettel) [inline .travis.yml]: Hm.  Now I will only ever get untested PRs.  Not sure I like that for the longer term.   Ok for now/today.

Comment (csgillespie) [reply .travis.yml]: I think the "proper" solution would be for you to add the API key as an encrypted travis environment variable. Then create the json file on travis. A similar idea to the drat/travis stuff

Comment (eddelbuettel) [reply .travis.yml]: Could you take the lead on that, "copious spare time" permitting ?

**Pull request merged**